pyproject.toml

```
@@ -5,7 +5,7 @@ build-backend = "setuptools.build_meta"
 [project]
 
 name = "ghga_validator"
-version = "1.0.0"
+version = "1.0.1"
```

```suggestion
version = "1.1.0"
```
I suggest bump the minor version because it will no longer work together with other libraries if they have pydantic 1.